src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -487,6 +540,18 @@ public static ApplicationModel.IResourceBuilder<TDestination> WithReference<TDes
 
         public static ApplicationModel.IResourceBuilder<T> WithRelationship<T>(this ApplicationModel.IResourceBuilder<T> builder, ApplicationModel.IResource resource, string type)
             where T : ApplicationModel.IResource { throw null; }
+
+        public static ApplicationModel.IResourceBuilder<T> WithUrl<T>(this ApplicationModel.IResourceBuilder<T> builder, string url, string? displayText = null)
```

Comment:
This just adds a single url?

Reply:
Yes

Reply:
Does it replace the existing ones? Or just adds one when you call it over and over.

Reply:
Just adds one each time you call it